Website/DesktopModules/Hotcakes/Providers/DataProviders/SqlDataProvider/03.09.01.SqlDataProvider

```diff
@@ -0,0 +1,15 @@
+﻿ALTER TABLE {databaseOwner}[{objectQualifier}hcc_Product] ADD
```

Comment:
@ArielBlanco1990 When we write these scripts, they should be written in such a way that it assumes someone might run it a second time.  So, these statements should each first check to see if the column exists.  If it doesn't exist, create it.

Reply:
@WillStrohl This is now resolved.